src/bundles/curves/index.ts

```diff
@@ -5,7 +5,7 @@ import {
   CurveTransformer,
   Point,
 } from './types';
-import generateCurve from './webGL_Curves';
+import generateCurve from './curves_webgl';

```

Comment:
Naming convention for `.ts` files should be in camelcase (ie. `curvesWebGL` or `curvesWebgl`)